### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kkuehl**
Update README.md.
Include Q/A on how to triage a crash test.
Remove some funfuzz references.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,31 +73,13 @@ especially after updating major/minor OS versions. This sometimes manifests on M
...
 
-## ~~Running funfuzz~~
+## Running domfuzz
 
 *For the maintained version of jsfunfuzz, head to the [funfuzz](https://github.com/MozillaSecurity/funfuzz) repository.*
 
-~~To run **all of the domfuzz and js fuzzers** which test builds every 8 hours:~~
 
-`python -u funfuzz/loopBot.py -b "--random" --target-time 28800 | tee ~/log-loopBotPy.txt`
+`python ./domfuzz/dom/automation/loopdomfuzz.py --submit ~/build`
```
Comment (jschwartzentruber) [inline README.md]: I don't think we want `--submit` in the simplest use case here. Use with FuzzManager should be documented, but that can come later.

File: README.md [hunk 2]
```diff
@@ -138,6 +120,21 @@ Support for the following operating systems **have been removed**:
...
+**Q:** What version of Python does domfuzz require?
 
 **A:** We recommend the Python 2.7.x series. There is no support for Python3 yet.
+
+**Q:** How do I reproduce a crash from a test case?
+
+**A:** The domfuzz way:
+```bash
+ python ./domfuzz/dom/automation/domInteresting.py ~/build ~/test_case.html
+ ```
```
Comment (jschwartzentruber) [inline README.md]: You can format code without \`\`\` by surrounding with blank lines and indenting 4 spaces.

File: README.md [hunk 3]
```diff
@@ -138,6 +120,21 @@ Support for the following operating systems **have been removed**:
...
+ The manual way:<br>
```
Comment (jschwartzentruber) [inline README.md]: Ditto re: \`\`\` and then \<br\> won't be necessary.

File: README.md [hunk 4]
```diff
@@ -138,6 +120,21 @@ Support for the following operating systems **have been removed**:
...
+ ```bash
+mkdir -p ~/px/df/extensions
+cp ./domfuzz/dom/automation/constant-prefs.js ~/px/df/prefs.js
+echo ./domfuzz/domfuzz/dom/extension/ > ~/px/df/extensions/domfuzz@squarefree.com
+lldb -- ~/build/dist/bin/firefox -profile ~/px/df/ -no-remote ~/test_case.html
```
Comment (jschwartzentruber) [inline README.md]: Can you replace `~/px/df` with just `tmp_profile`?

**Review by jschwartzentruber — CHANGES_REQUESTED** [review]
Thanks for contributing!

**Comment by kkuehl** [comment]
Performed requested changes. Thanks!

**Pull request merged**